cmd/namespace_registry.go

```diff
@@ -1,206 +1,49 @@
+/*
```

Comment:
Remove the copyright statement for now.

IIUC, the copyright goes to Morgridge... to do this in bulk, I created a separate issue on the topic: https://github.com/PelicanPlatform/pelican/issues/42